icons/steadybit.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.0" viewBox="0 0 24 24"><path d="M23.7,12.6c-0.6-1.4-2.2-2.2-3.7-1.9l-2.9-4.7c1.1-1,1.4-2.6,0.8-3.9c-0.6-1.2-1.7-2-3-2c-0.5,0-1,0.1-1.4,0.3C12,1,11.2,2.6,11.5,4.1l-5.7,3c-1-1-2.6-1.4-3.9-0.8C1.1,6.8,0.5,7.4,0.2,8.3c-0.3,0.8-0.3,1.8,0.1,2.6c0.7,1.5,2.3,2.3,3.9,1.8l2.9,5.5c-1,1-1.2,2.5-0.6,3.8c0.5,1.2,1.7,2,3,2c0.5,0,1-0.1,1.4-0.3c1.4-0.6,2.1-2.2,1.9-3.7l5.3-3.9c0.9,1.1,2.6,1.6,4,0.9c0.8-0.4,1.4-1.1,1.7-1.9C24.1,14.3,24.1,13.4,23.7,12.6z M4.4,9.8c-0.1,0.3-0.3,0.5-0.6,0.6c-0.5,0.2-1.2,0-1.4-0.5c-0.1-0.3-0.1-0.6,0-0.8c0.1-0.3,0.3-0.5,0.6-0.6C3,8.4,3.2,8.4,3.4,8.4c0.4,0,0.8,0.2,1,0.6C4.5,9.3,4.5,9.6,4.4,9.8z M17.3,14.5L12,18.4c-0.2-0.2-0.4-0.4-0.7-0.6l3.4-10.9c0.3,0,0.5,0,0.8-0.1l2.9,4.7C17.6,12.2,17.2,13.4,17.3,14.5z M6.6,8.7l5.7-3c0.2,0.2,0.4,0.4,0.6,0.5L9.6,17.2c-0.3,0-0.7,0-1,0.1l-2.9-5.5C6.5,11,6.9,9.9,6.6,8.7z M15.8,3.8c-0.1,0.3-0.3,0.5-0.6,0.6c-0.5,0.2-1.2,0-1.4-0.5c-0.1-0.3-0.1-0.6,0-0.8c0.1-0.3,0.3-0.5,0.6-0.6c0.5-0.3,1.2,0,1.4,0.5C15.9,3.2,15.9,3.5,15.8,3.8zM8.5,20.2c0.1-0.3,0.3-0.5,0.6-0.6c0.1-0.1,0.3-0.1,0.5-0.1c0.4,0,0.8,0.2,1,0.6c0.1,0.3,0.1,0.6,0,0.8c-0.1,0.3-0.3,0.5-0.6,0.6c-0.5,0.3-1.2,0-1.4-0.5C8.4,20.8,8.4,20.5,8.5,20.2z M21.7,14.4c-0.1,0.3-0.3,0.5-0.6,0.6c-0.5,0.3-1.2,0-1.4-0.5c-0.1-0.3-0.1-0.6,0-0.8c0.1-0.3,0.3-0.5,0.6-0.6c0.1-0.1,0.3-0.1,0.5-0.1c0.4,0,0.8,0.2,1,0.6C21.8,13.8,21.8,14.1,21.7,14.4z" fill="#e53935"/></svg>
```

Comment:
The icon looks quite nice, but could you please make sure that you use one of the [Material Design Colors](https://pkief.github.io/material-color-converter/)? :)

Reply:
Sorry, the color is already correct, I saw it wrong when I looked over it 👍